wordpress-mobile/gutenberg-mobile
# File Block basics

Implement changes from Gutenberg: https://github.com/WordPress/gutenberg/pull/26278
WPiOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/15122
WPAndroid PR: https://github.com/wordpress-mobile/WordPress-Android/pull/13178

To test:
- Please check [Gutenberg](WordPress/gutenberg#26278) - [WPiOS](https://github.com/wordpress-mobile/WordPress-iOS/pull/15122) - [WPAndroid](https://github.com/wordpress-mobile/WordPress-Android/pull/13178) for testing instructions.

PR submission checklist:
